receiver/vcenterreceiver/metadata.yaml

```diff
@@ -0,0 +1,533 @@
+  vcenter.host.vsan.outstanding_io:
+    enabled: true
+    description: Outstanding IO from all vSAN clients in the host, such as virtual machines, stats object, etc.
+    unit: "{operations/sec}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.resource_pool.memory.usage:
+    enabled: true
+    description: The usage of the memory by the resource pool
+    unit: MBy
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.resource_pool.memory.shares:
+    enabled: true
+    description: The amount of shares of memory in the resource pool
+    unit: "{shares}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.resource_pool.cpu.usage:
+    enabled: true
+    description: The usage of the CPU used by the resource pool
+    unit: "{MHz}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.resource_pool.cpu.shares:
+    enabled: true
+    description: The amount of shares of CPU in the resource pool
+    unit: "{shares}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.vm.memory.ballooned:
+    enabled: true
+    description: The amount of memory that is ballooned due to virtualization.
+    unit: By
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.vm.memory.usage:
+    enabled: true
+    description: The amount of memory that is used by the virtual machine
+    unit: MBy
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.vm.cpu.utilization:
+    enabled: true
+    description: The CPU utilization of the virtual machine
+    unit: "%"
+    gauge:
+      value_type: double
+    attributes: []
+  vcenter.vm.disk.usage:
+    enabled: true
+    description: The amount of storage space the virtual machine is using
+    unit: By
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: [disk_state]
+  vcenter.vm.disk.utilization:
+    enabled: true
+    description: The utilization of storage on the virtual machine
+    unit: "%"
+    gauge:
+      value_type: double
+    attributes: []
+  vcenter.vm.disk.latency.avg:
+    enabled: true
+    description: The latency of operations to the virtual machine's disk
+    unit: µs
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: [latency_direction]
+    extended_documentation: This latency is the sum of the device and kernel write latencies. Requires Performance Counter level 2 for metric to populate.
+  vcenter.vm.disk.latency.max:
+    enabled: true
+    description: The highest reported total latency (device and kernel times)
```

Comment:
Since startup? Or is there an interval?

Reply:
Added a slight description update to help accommodate this!